client/src/components/main/shared/StrandSwitch/StrandSwitch.tsx

```diff
@@ -37,7 +37,7 @@ export default function StrandSwitch(
           checked={selectedStrand === "-"}
           icon={<AddCircleIcon color="primary" />}
           checkedIcon={<RemoveCircleIcon color="primary" />}
-          disableRipple
+          disabled={true}
```

Comment:
could just do shorthand here
```suggestion
          disabled
```